### ISSUE TITLE
The calendar functionality on the  Recent Revisions page is not working properly

### ISSUE THREAD

**Issue by ovidiuboca1** [mozilla/sumo-project#243]
STR:

1. Go to https://test.sumo.mozit.cloud/en-US/kb/revisions?locale=ro&users=&start=&end=
2. Click on the "Start" box
3. Try to select a different month or year.

ER: You can select a different year or month.
AR: You can't select a different year or month.

**Comment by kkellydesign** [comment]
This is working for me locally, but it's very slow on the test server. I think the call to the server may be timing out, because the waiting icon just hangs there. My local instance database is much smaller, so that's all happening a bit quicker. 

I did find some more visual bugs while testing this out though, and those are now fixed. 

<img width="336" alt="Screen Shot 2020-03-13 at 2 55 00 PM" src="https://user-images.githubusercontent.com/202590/76651329-a9d46900-653a-11ea-8413-2c883c5445bc.png">
